src/popover/Popover.js

```diff
@@ -223,7 +223,7 @@ const Popover = React.createClass({
   },
 
   isAbsolute() {
-    return this.getPopoverProps().type === 'absolute';
+    return this.getPopoverProps().attachToBody === true;
```

Comment:
You can resolve by using `!!` operator, like `!!this.getPopoverProps().attachToBody`. It always returns a boolean ;)